ZarCare Automation/Test/PageValidations/RegisterPageValidation.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
Please add all this libraries in using file

Reply:
Added to Using file